relay-server/src/envelope.rs

```diff
@@ -743,13 +752,14 @@ impl Item {
             ItemType::RawSecurity => true,
             ItemType::UnrealReport => true,
             ItemType::UserReport => true,
-            ItemType::ReplayEvent => true,
+            ItemType::ReplayEvent => false,
```

Comment:
>ReplayEvent can no longer "create an event" as this was putting it in a separate envelope for processing. The Envelope containing a replay should always contain an event_id, so this shouldn't be a problem.

Should we expect older SDKs to continue sending `ReplayEvent`s? This change seems like a breaking change for users not upgrading their SDKs.

Reply:
I don't believe this is a breaking change, as ReplayEvents and ReplayRecordings are always in the same envelope, which has an event_id envelope header.